.github/pull-request-template.md

```diff
@@ -1,4 +1,22 @@
-Fixes #(issue-number)
+<!-- PR tips
+
+New contributors:
+
+If you are new to Git/GitHub and want to make a quick fix to the docs,
+open your PR against the release branch where you found the error, such as
+"release-0.5".
+
+Experienced contributors:
+
+Most PRs should be opened against the master branch.
+
+If the change should also be in the most recent numbered release, add the
+corresponding "cherrypick-0.X" label; for example, "cherrypick-0.5." Best
+practice is to open a PR for the cherry-pick yourself after your PR has been
```

Comment:
```suggestion
practice is to open a subsequent pull request, PR-cherrypick, using the merged commit number of PR-original.
```

Reply:
I don't think this level of detail belongs in the note. I opened issue #1136 to add information about cherry-picking PRs back into releases that includes where to get the commit number from.